drud/ddev-ui
# add scss support

## The Problem/Issue/Bug:
Previously, only standard css was supported. With a bootstrap/scss styleguide coming, preparations needed to be made to integrate the new designs

## How this PR Solves The Problem:
With the introduction of webpack, we can now use compass to automatically transpile SCSS to CSS

## Manual Testing Instructions:
- run make build to generate new package.json
- run npm start to launch hotreload server and application
- edit sass/base.scss and adjust the background-color hex to anything you want and save the change
- Notice that the app instantly updated with the new background color

## Related Issue Link(s):
#4
